firelyserver/releasenotes/releasenotes.rst

```diff
@@ -13,10 +13,91 @@ Release notes Firely Server
 .. _upgrade_vonk:
 
 Upgrading Firely Server
---------------
+-----------------------
 
 See :ref:`upgrade` for information on how to upgrade to a new version of Firely Server.
 
+.. _vonk_releasenotes_400:
+
+Release 4.0.0
+-------------
+
+This major version introduces a new name: **Firely Server** instead of Vonk. Other than that, this release contains some significant code changes, which could impact you if you run Firely Server with your own plugins.
+
+Features
+^^^^^^^^
+
+#. Name change Vonk -> Firely Server:
+   #. The main entry point dll (formerly: ``Vonk.Server.dll``) and executable (formerly: ``Vonk.Server.exe``) names have been changed to ``Firely.Server.dll`` and ``Firely.Server.exe`` respectively.
```

Comment:
The default license file was also renamed to `firelyserver-license.json`, which will break if you used the old vonk-license.json name.